### PR TITLE
dm/doc

### PR DESCRIPTION
**Pull request by dmerejkowsky**

### CODE REVIEW

**Comment by cgestes** [comment]
cool. maybe add a warning about quote in the shell?

or examples...

cause it is "\$1" and '\$1'

**Comment by dmerejkowsky** [comment]
> cool. maybe add a warning about quote in the shell?
Done in the README.

I've also added an 'EXAMPLES' section at the end of the `--help` message

File: README.md [hunk 1]
```diff
@@ -58,21 +58,48 @@ Patching src/top.txt
...
+
+
+If you don't want the pattern to be used as a regex, use the `--no-regex` command line flag.
+
+This makes it possible to look for special characters without escaping them:
 
 ```
-$ ruplacer '(\d{2})/(\d{2})/(\d{4})' '$3-$1-$2'
+# This is a regex that matches the letter a
+# or the letter u
```
Comment (blastrock) [inline README.md]: *or the letter o

File: README.md [hunk 2]
```diff
@@ -58,21 +58,48 @@ Patching src/top.txt
...
+$ ruplacer (a|o) u
```
Comment (blastrock) [inline README.md]: put single quotes, i don't think my shell would like this line, same for the one below

**Pull request merged**